scripts/griddle.jsx

```diff
@@ -707,11 +708,11 @@ var Griddle = React.createClass({
                 hasMorePages={hasMorePages}
                 onRowClick={this.props.onRowClick}/></div>)
     },
-    getContentSection: function(data, cols, meta, pagingContent, hasMorePages){
+    getContentSection: function(data, extraData, cols, meta, pagingContent, hasMorePages){
```

Comment:
Should extraData be the last parameter in all of the function calls or just in `getContentSection`?